desktop-src/WES/defining-keywords-used-to-classify-types-of-events.md

```diff
@@ -14,22 +14,25 @@ An ETW tracing session can use the keywords (in the same way that it uses level)
 
 For example, if the provider defines an event that specifies a read keyword (bit 0) and a local access keyword (bit 1), and a second event that specifies a read keyword (bit 0) and a remote access keyword (bit 2), you could set the "Any" bitmask to 1 to receive all read events, or you could set the "Any" bitmask to 1 and "All" bitmask to 3 to receive only local reads.
 
-The following example shows how to define a keyword. You must specify the keyword's **name** and **mask** attributes. The mask must have only one set—the keyword identifies the bit. The event definition's **keyword** attribute contains a space-separated list of keyword names. The **symbol** and **message** attributes are optional.
+You must specify the keyword's **name** and **mask** attributes. The mask must set only one bit, between bit 0 and bit 47. Bits 48 through 64 are reserved by the system. If an event's **keyword** attribute contains a space-separated list of keyword names then the event's keyword mask will be each of specified keywords' masks OR'ed together.
```

Comment:
"reserved by the system" - maybe better to say "have predefined semantics as described in winmeta.xml".

"will be each of specified keywords' masks" - missing a "the", and somewhat awkward to parse. Not sure we even need to say this.